.travis.yml

```diff
@@ -19,6 +19,7 @@ before_script:
                 eastwood "{:exclude-namespaces [cloverage.sample.exercise-instrumentation]}", \
                 eastwood "{:namespaces [cloverage.sample.exercise-instrumentation] \
                            :exclude-linters [:unused-ret-vals \
+                                             :suspicious-expression \
```

Comment:
Eastwood was complaining about the `(+ 1)` forms I added to `cloverage.sample.exercise-instrumentation` for the tests